src/interface/TWAnyAddress.cpp

```diff
@@ -90,6 +90,16 @@ TWData* _Nonnull TWAnyAddressData(struct TWAnyAddress* _Nonnull address) {
         break;
     }
 
+    case TWCoinTypeCryptoorg:
```

Comment:
merge this into Cosmos case

Reply:
Good point, leftover from when Address class was duplicated.